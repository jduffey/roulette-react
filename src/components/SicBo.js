export function SicBo() {
    return (
        <div
            style={{
                color: "white",
                fontSize: "4rem",
            }}
        >
            PLACEHOLDER PAGE FOR SIC BO
        </div>
    );
}
