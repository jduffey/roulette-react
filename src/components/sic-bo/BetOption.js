// import { Chip } from "./Chip";

const BETTING_SQUARE_OUTLINE_SIZE = 1;

const CLASS_NAME = "BetOption-component";
export function BetOption(props) {
    return (
        <div
            className={CLASS_NAME}
            onClick={props.onClick}
            style={{
                left: props.styleData.left,
                top: props.styleData.top,
                width: props.styleData.width - BETTING_SQUARE_OUTLINE_SIZE,
                height: props.styleData.height - BETTING_SQUARE_OUTLINE_SIZE,
                backgroundColor: props.styleData.backgroundColor,
            }}
        >
            <div className="betting-square-contents">
                <div
                    className={`betting-square-label`}
                    style={{
                        backgroundColor: props.styleData.labelBackgroundColor,
                    }}
                >
                    {props.displayText}
                </div>
            </div>
        </div>
    );
}
