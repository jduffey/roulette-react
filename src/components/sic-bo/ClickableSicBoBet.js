import { BET_NAMES } from "./sicBoBetNames";
import { STANDARD_COLORS } from "../../common/standardColors";

const BETTING_SQUARE_OUTLINE_SIZE = 1;

const CLASS_NAME = "ClickableSicBoBet-component";
function ClickableSicBoBet(props) {
    return (
        <div
            className={CLASS_NAME}
            onClick={props.onClick}
            style={{
                left: props.styleData.left,
                top: props.styleData.top,
                width: props.styleData.width - BETTING_SQUARE_OUTLINE_SIZE,
                height: props.styleData.height - BETTING_SQUARE_OUTLINE_SIZE,
                backgroundColor: props.styleData.backgroundColor,
            }}
        >
            <div className="betting-square-contents">
                <div
                    className={`betting-square-label`}
                    style={{
                        backgroundColor: props.styleData.labelBackgroundColor,
                    }}
                >
                    {props.displayText}
                </div>
            </div>
        </div>
    );
}

const BET_OPTION_DISPLAY_PARAMS = Object.entries({
    [BET_NAMES.SMALL]: [0, 60, 200, 160, STANDARD_COLORS.INHERIT, "Small!"],
    [BET_NAMES.BIG]: [1200, 60, 200, 160, STANDARD_COLORS.INHERIT, "Big!"],
    [BET_NAMES.DOUBLE_ONE]: [210, 100, 60, 120, STANDARD_COLORS.INHERIT, "1 1"],
    [BET_NAMES.DOUBLE_TWO]: [280, 100, 60, 120, STANDARD_COLORS.INHERIT, "2 2"],
    [BET_NAMES.DOUBLE_THREE]: [350, 100, 60, 120, STANDARD_COLORS.INHERIT, "3 3"],
    [BET_NAMES.DOUBLE_FOUR]: [990, 100, 60, 120, STANDARD_COLORS.INHERIT, "4 4"],
    [BET_NAMES.DOUBLE_FIVE]: [1060, 100, 60, 120, STANDARD_COLORS.INHERIT, "5 5"],
    [BET_NAMES.DOUBLE_SIX]: [1130, 100, 60, 120, STANDARD_COLORS.INHERIT, "6 6"],
    [BET_NAMES.TRIPLE_ONE]: [420, 100, 120, 40, STANDARD_COLORS.INHERIT, "1 1 1"],
    [BET_NAMES.TRIPLE_TWO]: [420, 140, 120, 40, STANDARD_COLORS.INHERIT, "2 2 2"],
    [BET_NAMES.TRIPLE_THREE]: [420, 180, 120, 40, STANDARD_COLORS.INHERIT, "3 3 3"],
    [BET_NAMES.TRIPLE_FOUR]: [860, 100, 120, 40, STANDARD_COLORS.INHERIT, "4 4 4"],
    [BET_NAMES.TRIPLE_FIVE]: [860, 140, 120, 40, STANDARD_COLORS.INHERIT, "5 5 5"],
    [BET_NAMES.TRIPLE_SIX]: [860, 180, 120, 40, STANDARD_COLORS.INHERIT, "6 6 6"],
    [BET_NAMES.ANY_TRIPLE]: [550, 100, 300, 120, STANDARD_COLORS.INHERIT, "Any Triple"],
    [BET_NAMES.SUM_4]: [0, 220, 100, 160, STANDARD_COLORS.INHERIT, "4"],
    [BET_NAMES.SUM_5]: [100, 220, 100, 160, STANDARD_COLORS.INHERIT, "5"],
    [BET_NAMES.SUM_6]: [200, 220, 100, 160, STANDARD_COLORS.INHERIT, "6"],
    [BET_NAMES.SUM_7]: [300, 220, 100, 160, STANDARD_COLORS.INHERIT, "7"],
    [BET_NAMES.SUM_8]: [400, 220, 100, 160, STANDARD_COLORS.INHERIT, "8"],
    [BET_NAMES.SUM_9]: [500, 220, 100, 160, STANDARD_COLORS.INHERIT, "9"],
    [BET_NAMES.SUM_10]: [600, 220, 100, 160, STANDARD_COLORS.INHERIT, "10"],
    [BET_NAMES.SUM_11]: [700, 220, 100, 160, STANDARD_COLORS.INHERIT, "11"],
    [BET_NAMES.SUM_12]: [800, 220, 100, 160, STANDARD_COLORS.INHERIT, "12"],
    [BET_NAMES.SUM_13]: [900, 220, 100, 160, STANDARD_COLORS.INHERIT, "13"],
    [BET_NAMES.SUM_14]: [1000, 220, 100, 160, STANDARD_COLORS.INHERIT, "14"],
    [BET_NAMES.SUM_15]: [1100, 220, 100, 160, STANDARD_COLORS.INHERIT, "15"],
    [BET_NAMES.SUM_16]: [1200, 220, 100, 160, STANDARD_COLORS.INHERIT, "16"],
    [BET_NAMES.SUM_17]: [1300, 220, 100, 160, STANDARD_COLORS.INHERIT, "17"],
    [BET_NAMES.PAIR1_2]: [200, 380, 80, 160, STANDARD_COLORS.INHERIT, "1 & 2"],
    [BET_NAMES.PAIR1_3]: [280, 380, 80, 160, STANDARD_COLORS.INHERIT, "1 & 3"],
    [BET_NAMES.PAIR1_4]: [360, 380, 80, 160, STANDARD_COLORS.INHERIT, "1 & 4"],
    [BET_NAMES.PAIR1_5]: [440, 380, 80, 160, STANDARD_COLORS.INHERIT, "1 & 5"],
    [BET_NAMES.PAIR1_6]: [520, 380, 80, 160, STANDARD_COLORS.INHERIT, "1 & 6"],
    [BET_NAMES.PAIR2_3]: [600, 380, 80, 160, STANDARD_COLORS.INHERIT, "2 & 3"],
    [BET_NAMES.PAIR2_4]: [680, 380, 80, 160, STANDARD_COLORS.INHERIT, "2 & 4"],
    [BET_NAMES.PAIR2_5]: [760, 380, 80, 160, STANDARD_COLORS.INHERIT, "2 & 5"],
    [BET_NAMES.PAIR2_6]: [840, 380, 80, 160, STANDARD_COLORS.INHERIT, "2 & 6"],
    [BET_NAMES.PAIR3_4]: [920, 380, 80, 160, STANDARD_COLORS.INHERIT, "3 & 4"],
    [BET_NAMES.PAIR3_5]: [1000, 380, 80, 160, STANDARD_COLORS.INHERIT, "3 & 5"],
    [BET_NAMES.PAIR3_6]: [1080, 380, 80, 160, STANDARD_COLORS.INHERIT, "3 & 6"],
    [BET_NAMES.PAIR4_5]: [1160, 380, 80, 160, STANDARD_COLORS.INHERIT, "4 & 5"],
    [BET_NAMES.PAIR4_6]: [1240, 380, 80, 160, STANDARD_COLORS.INHERIT, "4 & 6"],
    [BET_NAMES.PAIR5_6]: [1320, 380, 80, 160, STANDARD_COLORS.INHERIT, "5 & 6"],
    [BET_NAMES.ONE]: [0, 540, 220, 80, STANDARD_COLORS.INHERIT, "1"],
    [BET_NAMES.TWO]: [230, 540, 220, 80, STANDARD_COLORS.INHERIT, "2"],
    [BET_NAMES.THREE]: [460, 540, 220, 80, STANDARD_COLORS.INHERIT, "3"],
    [BET_NAMES.FOUR]: [690, 540, 220, 80, STANDARD_COLORS.INHERIT, "4"],
    [BET_NAMES.FIVE]: [920, 540, 220, 80, STANDARD_COLORS.INHERIT, "5"],
    [BET_NAMES.SIX]: [1180, 540, 220, 80, STANDARD_COLORS.INHERIT, "6"],
}).reduce((acc, [betOptionName, params]) => {
    acc[betOptionName] = {
        styleData: {
            left: params[0],
            top: params[1],
            width: params[2],
            height: params[3],
            backgroundColor: params[4],
        },
        displayText: params[5],
    };
    return acc;
}, {});

export { ClickableSicBoBet, BET_OPTION_DISPLAY_PARAMS };