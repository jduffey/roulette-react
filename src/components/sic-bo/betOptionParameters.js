import { BET_NAMES } from "./sicBoBetNames";
import { STANDARD_COLORS } from "../../common/standardColors";

export const BET_OPTION_PARAMS = Object.entries({
    [BET_NAMES.SMALL]: [0, 60, 200, 160, STANDARD_COLORS.INHERIT, "Small!"],
    [BET_NAMES.BIG]: [1200, 60, 200, 160, STANDARD_COLORS.INHERIT, "Big!"],
    [BET_NAMES.DOUBLE_ONE]: [210, 100, 60, 120, STANDARD_COLORS.INHERIT, "1 1"],
    [BET_NAMES.DOUBLE_TWO]: [280, 100, 60, 120, STANDARD_COLORS.INHERIT, "2 2"],
    [BET_NAMES.DOUBLE_THREE]: [350, 100, 60, 120, STANDARD_COLORS.INHERIT, "3 3"],
    [BET_NAMES.DOUBLE_FOUR]: [990, 100, 60, 120, STANDARD_COLORS.INHERIT, "4 4"],
    [BET_NAMES.DOUBLE_FIVE]: [1060, 100, 60, 120, STANDARD_COLORS.INHERIT, "5 5"],
    [BET_NAMES.DOUBLE_SIX]: [1130, 100, 60, 120, STANDARD_COLORS.INHERIT, "6 6"],
    [BET_NAMES.TRIPLE_ONE]: [420, 100, 120, 40, STANDARD_COLORS.INHERIT, "1 1 1"],
    [BET_NAMES.TRIPLE_TWO]: [420, 140, 120, 40, STANDARD_COLORS.INHERIT, "2 2 2"],
    [BET_NAMES.TRIPLE_THREE]: [420, 180, 120, 40, STANDARD_COLORS.INHERIT, "3 3 3"],
    [BET_NAMES.TRIPLE_FOUR]: [860, 100, 120, 40, STANDARD_COLORS.INHERIT, "4 4 4"],
    [BET_NAMES.TRIPLE_FIVE]: [860, 140, 120, 40, STANDARD_COLORS.INHERIT, "5 5 5"],
    [BET_NAMES.TRIPLE_SIX]: [860, 180, 120, 40, STANDARD_COLORS.INHERIT, "6 6 6"],
    [BET_NAMES.ANY_TRIPLE]: [550, 100, 300, 120, STANDARD_COLORS.INHERIT, "Any Triple"],
}).reduce((acc, [betOptionName, params]) => {
    acc[betOptionName] = {
        styleData: {
            left: params[0],
            top: params[1],
            width: params[2],
            height: params[3],
            backgroundColor: params[4],
            // labelBackgroundColor: params[5],
        },
        displayText: params[5],
    };
    return acc;
}, {});
