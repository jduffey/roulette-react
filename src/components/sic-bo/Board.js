import React from 'react';

import { ClickableSicBoBet, BET_OPTION_DISPLAY_PARAMS } from './ClickableSicBoBet';

export function Board(props) {
    return (
        <div>
            {Object.entries(BET_OPTION_DISPLAY_PARAMS).map(([betOptionName, params]) => {
                return (
                    <ClickableSicBoBet
                        key={betOptionName}
                        displayText={params.displayText}
                        styleData={params.styleData}
                    />
                );
            })}
        </div>
    );
}
