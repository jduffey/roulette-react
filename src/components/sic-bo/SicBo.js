import { Board } from "./Board";

export function SicBo() {
    return (
        <Board />
    );
}
