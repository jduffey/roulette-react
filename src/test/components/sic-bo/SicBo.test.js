import renderer from 'react-test-renderer';

import { SicBo } from '../../../components/sic-bo/SicBo';

describe('SicBo', () => {
    it('renders', () => {
        const view = renderer.create(<SicBo />);
        expect(view).toMatchSnapshot();
    });
});
