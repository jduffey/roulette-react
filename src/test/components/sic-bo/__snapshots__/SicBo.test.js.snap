// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SicBo renders 1`] = `
Array [
  <div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 0,
          "top": 60,
          "width": 199,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          Small!
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 1200,
          "top": 60,
          "width": 199,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          Big!
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 119,
          "left": 210,
          "top": 100,
          "width": 59,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          1 1
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 119,
          "left": 280,
          "top": 100,
          "width": 59,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          2 2
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 119,
          "left": 350,
          "top": 100,
          "width": 59,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          3 3
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 119,
          "left": 990,
          "top": 100,
          "width": 59,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          4 4
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 119,
          "left": 1060,
          "top": 100,
          "width": 59,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          5 5
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 119,
          "left": 1130,
          "top": 100,
          "width": 59,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          6 6
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 39,
          "left": 420,
          "top": 100,
          "width": 119,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          1 1 1
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 39,
          "left": 420,
          "top": 140,
          "width": 119,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          2 2 2
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 39,
          "left": 420,
          "top": 180,
          "width": 119,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          3 3 3
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 39,
          "left": 860,
          "top": 100,
          "width": 119,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          4 4 4
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 39,
          "left": 860,
          "top": 140,
          "width": 119,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          5 5 5
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 39,
          "left": 860,
          "top": 180,
          "width": 119,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          6 6 6
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 119,
          "left": 550,
          "top": 100,
          "width": 299,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          Any Triple
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 0,
          "top": 220,
          "width": 99,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          4
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 100,
          "top": 220,
          "width": 99,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          5
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 200,
          "top": 220,
          "width": 99,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          6
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 300,
          "top": 220,
          "width": 99,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          7
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 400,
          "top": 220,
          "width": 99,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          8
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 500,
          "top": 220,
          "width": 99,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          9
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 600,
          "top": 220,
          "width": 99,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          10
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 700,
          "top": 220,
          "width": 99,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          11
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 800,
          "top": 220,
          "width": 99,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          12
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 900,
          "top": 220,
          "width": 99,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          13
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 1000,
          "top": 220,
          "width": 99,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          14
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 1100,
          "top": 220,
          "width": 99,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          15
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 1200,
          "top": 220,
          "width": 99,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          16
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 1300,
          "top": 220,
          "width": 99,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          17
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 200,
          "top": 380,
          "width": 79,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          1 & 2
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 280,
          "top": 380,
          "width": 79,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          1 & 3
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 360,
          "top": 380,
          "width": 79,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          1 & 4
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 440,
          "top": 380,
          "width": 79,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          1 & 5
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 520,
          "top": 380,
          "width": 79,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          1 & 6
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 600,
          "top": 380,
          "width": 79,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          2 & 3
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 680,
          "top": 380,
          "width": 79,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          2 & 4
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 760,
          "top": 380,
          "width": 79,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          2 & 5
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 840,
          "top": 380,
          "width": 79,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          2 & 6
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 920,
          "top": 380,
          "width": 79,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          3 & 4
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 1000,
          "top": 380,
          "width": 79,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          3 & 5
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 1080,
          "top": 380,
          "width": 79,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          3 & 6
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 1160,
          "top": 380,
          "width": 79,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          4 & 5
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 1240,
          "top": 380,
          "width": 79,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          4 & 6
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 159,
          "left": 1320,
          "top": 380,
          "width": 79,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          5 & 6
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 79,
          "left": 0,
          "top": 540,
          "width": 219,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          1
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 79,
          "left": 230,
          "top": 540,
          "width": 219,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          2
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 79,
          "left": 460,
          "top": 540,
          "width": 219,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          3
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 79,
          "left": 690,
          "top": 540,
          "width": 219,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          4
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 79,
          "left": 920,
          "top": 540,
          "width": 219,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          5
        </div>
      </div>
    </div>
    <div
      className="ClickableSicBoBet-component"
      style={
        Object {
          "backgroundColor": "inherit",
          "height": 79,
          "left": 1180,
          "top": 540,
          "width": 219,
        }
      }
    >
      <div
        className="betting-square-contents"
      >
        <div
          className="betting-square-label"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        >
          6
        </div>
      </div>
    </div>
  </div>,
  <div
    className="SicBo-PlayerInfo-component"
  >
    <div>
      Owned Bal.
      <br />
      10000
    </div>
    <div>
      Chips in Hand
      <br />
      10000
    </div>
    <div>
      Total Bet
      <br />
      0
    </div>
  </div>,
]
`;
